packages/flutter_tools/bin/xcode_backend.sh

```diff
@@ -54,64 +54,64 @@ BuildApp() {
     framework_path="${FLUTTER_FRAMEWORK_DIR}"
   fi
 
-  AssertExists ${project_path}
+  AssertExists "${project_path}"
 
-  local derived_dir=${SOURCE_ROOT}/Flutter
-  RunCommand mkdir -p $derived_dir
-  AssertExists $derived_dir
+  local derived_dir="${SOURCE_ROOT}/Flutter"
+  RunCommand mkdir -p "$derived_dir"
+  AssertExists "$derived_dir"
 
-  RunCommand rm -f ${derived_dir}/Flutter.framework
-  RunCommand rm -f ${derived_dir}/app.dylib
-  RunCommand rm -f ${derived_dir}/app.flx
-  RunCommand cp -r ${framework_path}/Flutter.framework ${derived_dir}
-  RunCommand pushd ${project_path}
+  RunCommand rm -f "${derived_dir}/Flutter.framework"
+  RunCommand rm -f "${derived_dir}/app.dylib"
+  RunCommand rm -f "${derived_dir}/app.flx"
+  RunCommand cp -r "${framework_path}/Flutter.framework ${derived_dir}"
```

Comment:
If you really want to be correct, you should add `--` to separate the flags from the file names